proto/commonfate/access/v1alpha1/grant.proto

```diff
@@ -56,14 +56,13 @@ enum GrantStatus {
     GRANT_STATUS_PENDING = 1;
 
 
-    GRANT_STATUS_PROVISIONING = 2;
 
     // The grant to the entitlement is currently active. 
-    GRANT_STATUS_ACTIVE = 3;
+    GRANT_STATUS_ACTIVE = 2;
```

Comment:
In proto bufs these numbers on the fields are designed to support backwards compatibility.
When you add or removes fiels you shoudl not be changing the numbers of the remaining fields.

This way, older clients can still read the new data correctly.

For example someone running an older version of the CLI will have issues with grant statuses because their client may interpret the grant status as the wrong value because its number has changed.

So when removing a field for this PR, you shoudl only remove teh field and leave everything else unchanged

Reply:
More info here in the spec https://protobuf.dev/programming-guides/proto3/#assigning